Update docs for asset limit (#24842)
cloudflare/cloudflare-docs: src/content/changelog/workers/2025-09-02-increased-static-asset-limits.mdx

---
title: Increased static asset limits for Workers
description: Paid and Workers for Platforms users can now upload up to 100,000 static assets per Worker version, up from the previous limit of 20,000.
products:
  - workers
  - workers-for-platforms
date: 2025-09-02
---

> Cloudflare has increased the static asset limits for Workers:

Reviewer: Use 2nd person rather than 3rd

```suggestion
You can now upload up to **100,000 static assets** per Worker version
```

- Paid and Workers for Platforms users can now upload up to **100,000 static assets** per Worker version, a 5x increase from the previous limit of 20,000.

> - Free users will continue to have a limit of 20,000 static assets per Worker version.

Reviewer: ```suggestion
- Customers on the free plan still have the same limit as before — 20,000 static assets per version of your Worker
```

> - The individual file size limit of 25 MiB remains unchanged for all users.

Reviewer: ```suggestion
- The individual file size limit of 25 MiB remains unchanged for all customers.
```

This increase allows you to build larger applications with more static assets without hitting limits.

### Wrangler

To take advantage of the increased limits, you must use **Wrangler version 4.34.0 or higher**.

> Earlier versions of Wrangler will continue to enforce the previous 20,000 file limit.

Reviewer: Why?
